tests/support.py

```diff
@@ -211,6 +212,17 @@ def cleanpath(relative_path, test_case):
     return tmp_path
 
 
+
+def fixturepath(relative_path):
+    tmp_path = os.path.join(TEST_DATA_DIR, relative_path)
+    assert (not stat.ISDIR(os.stat(tmp_path)))
+    return tmp_path
+
+
+def projectrelative(tmp_path):
```

Comment:
Minor detail, but you may want to reconsider the name here as we generally refer to `vgrids` in the `gdp_mode` context (SIF) when talking about `project`. I suppose `package` would be a valid replacement here if you don't have a better suggestion.

Reply:
Oh fair point - leave that with me to think of something.

Reply:
Addressed in the latest push - have made this a static method on the test case (see https://github.com/ucphhpc/migrid-sync/pull/66/files#diff-fdb3655f4432467b4782aa69fa9340872e0e2e90e0b9bc7ac370ef610d024911R210).